app/services/payout_report_publisher_includer.rb

```diff
@@ -10,40 +10,52 @@ def perform
     publisher_has_unsettled_balance = false
 
     wallet = @publisher.wallet
-    # If the wallet has been blocked from uphold's terms of service. In this state users are unable to login or access uphold.
-    return if wallet.nil? || wallet.blocked?
 
+    uphold_status = wallet.uphold_account_status
+    uphold_member = wallet.is_a_member?
+    reauthorization_needed = wallet.action == "re-authorize"
+    suspended = @publisher.suspended?
+
+    # Create potential payment for referrals
     probi = wallet.channel_balances[@publisher.owner_identifier].probi_before_fees # probi = balance
-    if probi.positive?
-      publisher_has_unsettled_balance = true
+    publisher_has_unsettled_balance = probi.positive? ? true : false
```

Comment:
`publisher_has_unsettled_balance = probi.positive?`

Reply:
Resolved in 023ddd4